Update garbage-collector.md (#34005)
dotnet/docs: docs/core/runtime-config/garbage-collector.md

> [!NOTE]
>

> > - These configurations are only read by the runtime when the GC is initialize (usually this means during the process startup time). So if you change an environment variable when a  process is already running, that change will not be reflected in that process. Settings that can be changed through APIs during at run time such as [latency level](../../standard/garbage-collection/latency.md), are omitted from this page.

Reviewer: ```suggestion
> - These configurations are only read by the runtime when the GC is initialized (usually this means during the process startup time). If you change an environment variable when a process is already running, the change won't be reflected in that process. Settings that can be changed through APIs at run time, such as [latency level](../../standard/garbage-collection/latency.md), are omitted from this page.
```

> > - Because GC is per process, it rarely ever makes sense to set these configurations on the machine level. For example, you would not want to set every .NET process to use Server GC, or the same heap hard limit on a machine.

Reviewer: ```suggestion
> - Because GC is per process, it rarely ever makes sense to set these configurations at the machine level. For example, you wouldn't want every .NET process on a machine to use server GC or the same heap hard limit.
```
